tools/docker/README.md

```diff
@@ -32,4 +79,5 @@ For bug fixes or small changes, increment the last number of a [SemVer](https://
 
 ## Uploading Docker Images
 
-New versions of Asterinas's Docker images are automatically uploaded to DockerHub through Github Actions. Simply submit your PR that updates Asterinas's Docker image for review. After getting the project maintainers' approval, the [Docker image building workflow](../../.github/workflows/docker_build.yml) will be started, building the new Docker image and pushing it to DockerHub.
\ No newline at end of file
+New versions of Asterinas's Docker images are automatically uploaded to DockerHub through Github Actions. Simply submit your PR that updates Asterinas's Docker image for review. After getting the project maintainers' approval, the [Docker image building workflow](../../.github/workflows/docker_build.yml) will be started, building the new Docker image and pushing it to DockerHub.
```

Comment:
Building and pushing TDX images should also be added to this `../../.github/workflows/docker_build.yml` file, so the image can be pushed to the asterinas docker repo.